### ISSUE TITLE
Team colors too strong

### ISSUE THREAD

**Issue by ChillerDragon** [teeworlds/teeworlds#1733]
I am not sure if this discussion is already closed or somewhere else.
But I still dislike the new team color saturation. It is too strong bright and neon.

I would love to see something more 0.6ish. More chill and soft c:

**Comment by Dune-jr** [comment]
I believe this was discussed a month ago: #1575 

@LordSk's proposal:
![image](https://user-images.githubusercontent.com/355114/49043528-a5f8c800-f1cb-11e8-8de8-35fa6aef88aa.png)

And it was fixed by #1606

**Comment by Zatline** [comment]
I think the saturation changes done in #1606 are fine and I personally prefer the overall more saturated look of 0.7 compared to like old 0.6 but I think there could be improvments still what I find more concerning currently is the way team colors are generated based on the colors picked before hand it sometimes rly doesn't quite work example; limekitty on @Dune-jr his mockup the whiskers there are bearly visable. :(

**Comment by LordSk** [comment]
I feel like we're going in circles. The "before" version mixed colors with a stronger emphasis on custom user colors (which made the whiskers pop a bit more) but it was disliked because they were not homogeneous enough. I like the current version. If you want your skin markings to be more visible in team mode, use more contrast to differentiate them.

**Comment by oy** [comment]
Maybe markings could be weighted a bit more?

**Comment by msiglreith** [comment]
I also prefer the old, less saturated colors (not only nostalgia 😄)
The blue colors are more difficult to see on dark backgrounds like ctf2 as the contrast between the outline is decreased now and the tees become harder to see IMO.
On the other side the red tees are too bright and strain the eyes (imagine 50min ctf2 game with a fixed tee centered on the screen).

Old colors for comparison: https://www.teeworlds.com/images/screens/screenshot_desert.png

**Comment by Dune-jr** [comment]
We already had this whole discussion. The issue was discussed in #1575, with mockups, then there was a PR. You didn't miss the discussion: you gave your input there, then we settled on something and the PR was merged.

Can we not re-open issues like that? Same with the chat color issue being re-opened after the PR (#1697)
Might be a bit of a bold opinion, but I'd like to see this kind of issues closed for good. Like @LordSk said:

> I feel like we're going in circles.
